devito/ops/operator.py

```diff
@@ -0,0 +1,21 @@
+from devito.ir.support import align_accesses
+from devito.operator import OperatorRunnable
+
+__all__ = ['Operator']
+
+
+class OperatorOPS(OperatorRunnable):
+
+    def _specialize_exprs(self, expressions):
```

Comment:
do you understand what this method does in ``core`` and why?

Reply:
here it's probably useless (so you can remove it) as:
1) AFAICT, it seems identical to the one in the superclass, so the same exact code would be executed
2) we are raising an exception anyway later on, so...

Reply:
Is this for transforming the IET?